src/backup_restore.cpp

```diff
@@ -128,10 +128,165 @@ void BackupRestore::backupAndRestoreIpzVpd(types::IPZVpdMap& io_srcVpdMap,
                                            const std::string& i_srcPath,
                                            const std::string& i_dstPath)
 {
-    (void)io_srcVpdMap;
-    (void)io_dstVpdMap;
-    (void)i_srcPath;
-    (void)i_dstPath;
+    if (!m_backupRestoreCfgJsonObj.contains("backupMap") ||
```

Comment:
This check was already done before calling this API. This is a private API so can be skipped.

Reply:
Updated